mediaapi/routing/download.go

```diff
@@ -369,8 +384,49 @@ func (r *downloadRequest) respondFromLocalFile(
 		" object-src 'self';"
 	w.Header().Set("Content-Security-Policy", contentSecurityPolicy)
```

Comment:
Wouldn't it be better to set these headers in the `!forFederation` block rather than deleting them in the federation block?

Reply:
Good point.